cmd/kyverno/main.go

```diff
@@ -126,18 +128,18 @@ func main() {
 		// in text mode we use FormatSerialize format
 		log.SetLogger(klogr.New())
 	} else if logFormat == "json" {
-		// TODO: klog does not support json out of the box, we need to provide a logger with json support to klog here
-		_, err := zap.NewProduction()
+		zapLog, err := zap.NewProduction()
 		if err != nil {
-			setupLog.Error(err, "Failed to initialize JSON logger")
+			fmt.Printf("failed to initialize JSON logger: %s", err.Error())
 			os.Exit(1)
 		}
-		// klog.SetLogger(zapr.NewLogger(zapLog))
+		klog.SetLogger(zapr.NewLogger(zapLog))
 
 		// in json mode we use FormatKlog format
 		log.SetLogger(klog.NewKlogr())
 	} else {
-		// TODO
+		fmt.Println("log format not recognized, pass `text` for text mode or `json` to enable JSON logging")
```

Comment:
@eddycharly what do you think about the info string

Reply:
Looks good!